client/layout/controller.js

```diff
@@ -121,28 +124,55 @@ class Controller extends Component {
 	}
 }
 
-// Update links in wp-admin menu to persist time related queries
-window.wpNavMenuUrlUpdate = function( page, query ) {
-	const search = stringifyQuery( getPersistedQuery( query ) );
-
-	Array.from(
-		document.querySelectorAll( `#${ page.wpOpenMenu } a, #${ page.wpClosedMenu } a` )
-	).forEach( item => {
-		/**
-		 * Example hrefs:
-		 *
-		 * http://example.com/wp-admin/admin.php?page=wc-admin#/analytics/orders?period=today&compare=previous_year
-		 * http://example.com/wp-admin/admin.php?page=wc-admin#/?period=week&compare=previous_year
-		 * http://example.com/wp-admin/admin.php?page=wc-admin
-		 */
-		if ( item.href.includes( 'wc-admin' ) && ! item.href.includes( 'devdocs' ) ) {
+/**
+ * Update an anchor's link in sidebar to include persisted queries. Leave excluded screens
+ * as is.
+ *
+ * @param {HTMLElement} item - Sidebar anchor link.
+ * @param {string} nextQuery - A query string to be added to updated hrefs.
+ * @param {Array} excludedScreens - wc-admin screens to avoid updating.
+ */
+export function updateLinkHref( item, nextQuery, excludedScreens ) {
+	/**
+	 * Regular expression with a lookbehind for "page=wc-admin", matching any WooCommerce Admin screen.
```

Comment:
@psealock should this comment be updated now that you removed the lookbehind?

Reply:
Fixed in https://github.com/woocommerce/woocommerce-admin/pull/2225/commits/a25746bb59480104abdd3360e0f5ef359820fd1d